clang/include/clang/Basic/BuiltinsRISCVCOREV.def

```diff
@@ -0,0 +1,163 @@
+//==- BuiltinsRISCVCOREV.def - RISC-V CORE-V Builtin database ----*- C++ -*-==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines the CORE-V-specific builtin function database.  Users of
+// this file must define the BUILTIN macro to make use of this information.
+//
+//===----------------------------------------------------------------------===//
+
+#if defined(BUILTIN) && !defined(TARGET_BUILTIN)
+#   define TARGET_BUILTIN(ID, TYPE, ATTRS, FEATURE) BUILTIN(ID, TYPE, ATTRS)
+#endif
+
+TARGET_BUILTIN(simd_add_h, "UZiUZiUZiIUc", "nc", "xcvsimd")
```

Comment:
Do all of these builtins actually demand `int32_t` as the types, even on `rv64`?

Reply:
Do you mean the scope of these builtins should be restricted to `xcvsimd,32bit`?

Reply:
This is resolved. It is a matter for when the 64-bit builtins are defined.